### PR TITLE
Fix main schema build errors and wrapping

### PR DESCRIPTION
**Pull request by LPardue**
- main: temporarily remove {: .language-json} tags until diagrams are changed
- main: wrap lines to appease xml2rfc

Helps address part of #188.

In this spec, you have a fundamental problem that the json in examples is malformed and kramdown complains. So let's just disable that for now. 

The rest of the changes are tedious but also not too onerous. The main issue you have is that there's lots of comments here and there that break length limits. So I've just picked a line length that will appease xml2rfc. 

In some cases, you have comments inside JSON, which won't work. In one case I deleted some because the qlog properties might get deleted anyway.

The changes you have in mind might have obviated the errors anyway. But if not, these changes will set a strong foundation to know how to shut xml2rfc up.

### CODE REVIEW

File: draft-ietf-quic-qlog-main-schema.md [hunk 1]
```diff
@@ -1477,11 +1481,10 @@ JSON-SEQ serialization example:
       }
     }
 }
-<RS>{"time": 2, "name": "transport:packet_received", "data": { ... } }
-<RS>{"time": 7, "name": "http:frame_parsed", "data": { ... } }
+<RS>{"time": 2, "name": "transport:parameters_set", "data": { ... } }
+<RS>{"time": 7, "name": "transport:packet_sent", "data": { ... } }
```
Comment (nibanks) [inline draft-ietf-quic-qlog-main-schema.md:1485]: Why these changes?

Comment (LPardue) [reply draft-ietf-quic-qlog-main-schema.md:1485]: in this specific case, the newlines matter. While the format does support them, it might confuse readers. So I changed the events to ones that had a slightly shorter name but still seemed to be semantically valid.

Comment (nibanks) [reply draft-ietf-quic-qlog-main-schema.md:1485]: Ok.

**Review by nibanks — APPROVED** [review]

**Review by rmarx — APPROVED** [review]
